arch/cortex-m/src/lib.rs

```diff
@@ -144,8 +141,8 @@ extern "C" {
     pub fn initialize_ram_jump_to_main();
 }
 
-#[cfg(all(target_arch = "arm", target_os = "none"))]
-global_asm!(
+#[cfg(any(doc, all(target_arch = "arm", target_os = "none")))]
+core::arch::global_asm!(
```

Comment:
Does this do anything?

Reply:
I don't think so. My intention here was to make every target selector also set `doc` properly so when someone copies this expression it would be correct.

Reply:
Ok that was my assumption.